src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -242,6 +246,28 @@ services.AddHttpClient(
 Note: The single instance returned by `HttpClientFactory` is reused by all
 export requests.
 
+## Experimental features
```

Comment:
@alanwest FYI I took this from your previous PR.

Reply:
categorize this into signals (logs, metrics sub-headings)